.github/workflows/bolt_cli_release_bins.yml

```diff
@@ -0,0 +1,141 @@
+name: Build and push Bolt CLI release binaries
+
+on:
+  push:
+    tags:
+      - "v*" # Push when version tags are pushed (e.g. v0.2.4)
+  release:
+    types:
+      - created # Push when a new release is created
+  workflow_dispatch: # allows manual triggering of the workflow
+
+env:
+  CARGO_TERM_COLOR: always
+  PROFILE: release
+
+jobs:
+  build-and-push:
+    name: ${{ matrix.target }} (${{ matrix.runner }})
+    runs-on: ${{ matrix.runner }}
+    timeout-minutes: 240
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          # `runner`: GHA runner label
+          # `target`: Rust build target triple
+          # `platform` and `arch`: Used in tarball names
+          - runner: Linux-24.04
+            target: x86_64-unknown-linux-gnu
+            platform: linux
+            arch: amd64
+          - runner: Linux-24.04
+            target: aarch64-unknown-linux-gnu
+            platform: linux
+            arch: arm64
+          - runner: macos-12-large
+            target: x86_64-apple-darwin
+            platform: darwin
+            arch: amd64
+          - runner: macos-latest-large
+            target: aarch64-apple-darwin
+            platform: darwin
+            arch: arm64
+          - runner: Windows
+            target: x86_64-pc-windows-msvc
+            platform: win32
+            arch: amd64
+
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v4
+
+      - name: Move to bolt-cli directory
+        run: |
+          cd bolt-cli
+
+      - name: Install Rust toolchain
+        uses: dtolnay/rust-toolchain@stable
+        with:
+          targets: ${{ matrix.target }}
+
+      - name: Install Cross
+        run: |
+          cargo install cross --force
+
+      #   - name: Install Protoc
+      #     uses: arduino/setup-protoc@v3
```

Comment:
remove or do you need this later?

Reply:
They might be needed if the workflow fails, but I don't know if it will, as I need to merge this to test it. 
I'll remove it in a further PR if everything is ok!